cinema/urls.py

```diff
@@ -0,0 +1,10 @@
+from django.urls import path
+
+from cinema.views import movie_list, movie_detail
+
+app_name = "cinema"
+
+urlpatterns = [
+    path("movies/", movie_list, name="bus_list"),
+    path("movies/<int:pk>/", movie_detail, name="bus_detail"),
```

Comment:
Similarly, the name for this path is set to 'bus_detail'. It should probably be 'movie_detail' to align with the view function and ensure consistency.